src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -239,6 +248,7 @@ private void GenerateGapTimeEntryBlocks(List<Toggl.TogglTimeEntryView> timeEntri
                         Started = prevEnd.Value + 1,
                         Ended = entry.Started - 1
                     };
+                    gaps.Add(block);
                     if (block.Height > 10) // Don't display to small gaps not to obstruct the view
                         gaps.Add(block);

```

Comment:
are you adding the same gap twice?

Reply:
Looks like faulty merge conflicts resolving, will fix this.